baronet2/boulder2vec
# Validation notebook with k-fold validation

Changes in this PR:

- I applied the k-fold to the full `df` without applying `train_test_split`. As we discussed on the call yesterday, I don't think we need train/val/test splits for this project.
- Simplified the model saving
- Added the validation notebook which loads all the models, calculates the metrics, and produces the plot

Test instructions:

- Run the `lr` and `pmf` scripts, and then run the eval notebook. The results should be identical to those in the PR.
